### PR TITLE
Ensure compliance with SAP Open Source Program

### PR DESCRIPTION
**Pull request by boyan-velinov**
Jira: LMCROSSITXSADEPLOY-2392

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+### How to contribute - the Process
+
+#### Fork the project
+* To develop your contribution to the project, first [fork](https://help.github.com/articles/fork-a-repo/) this repository in your own github account. 
+
+* When developing make sure to keep your fork up to date with the origin's master branch or the release branch you want to contribute a fix to.
+
+#### How to build and run?
+You can read how to build, configure and run the MultiApps Controller [here](https://github.com/SAP/cf-java-client-sap#compiling-and-packaging).
```
Comment (theghost5800) [inline CONTRIBUTING.md:112]: Wrong project name

File: README.md [hunk 2]
```diff
@@ -1,12 +1,17 @@
-# cf-java-client [![Build Status](https://travis-ci.org/SAP/cf-java-client-sap.svg?branch=master)](https://travis-ci.org/SAP/cf-java-client-sap) [![REUSE status](https://api.reuse.software/badge/github.com/SAP/cf-java-client-sap)](https://api.reuse.software/info/github.com/SAP/cf-java-client-sap)
+# Description [![Build Status](https://travis-ci.org/SAP/cf-java-client-sap.svg?branch=master)](https://travis-ci.org/SAP/cf-java-client-sap) [![REUSE status](https://api.reuse.software/badge/github.com/SAP/cf-java-client-sap)](https://api.reuse.software/info/github.com/SAP/cf-java-client-sap)
```
Comment (theghost5800) [inline README.md:1]: I think this link refer to old travis.

Comment (boyan-velinov) [reply README.md]: Nice catch

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+## Help Others
+
+You can help cf-java-client-sap by helping others who use it and need support.
+
+## Analyze Issues
+
+Analyzing issue reports can be a lot of effort. Any help is welcome!
+Go to [the Github issue tracker](https://github.com/SAP/cf-java-client-sap/issues?state=open) and find an open issue which needs additional work or a bugfix.
+
+Additional work may be further information, or a minimized jsbin example or gist, or it might be a hint that helps understanding the issue. Maybe you can even find and [contribute](#contribute-code) a bugfix?
```
[the remark this answers is not in the document]
Comment (boyan-velinov) [reply CONTRIBUTING.md]: Removing (leftover from copy-pasting)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+If the bug appears to be a regression introduced in a new version of the project, try to find the closest versions between which it was introduced and take special care to make sure the issue is not caused by your application's usage of any internal method which changed its behavior.
+
+
+### Reporting Security Issues
+
+We take security issues in our projects seriously. We appreciate your efforts to responsibly disclose your findings.
+
+Please do not report security issues directly on GitHub but using one of the channels listed below. This allows us to provide a fix before an issue can be exploited.
+
+- **Researchers/Non-SAP Customers:** Please consult SAPs [disclosure guidelines](https://wiki.scn.sap.com/wiki/display/PSR/Disclosure+Guidelines+for+SAP+Security+Advisories) and send the related information in a PGP encrypted e-mail to secure@sap.com. Find the public PGP key [here](https://www.sap.com/dmc/policies/pgp/keyblock.txt).
```
[the remark this answers is not in the document]
Comment (boyan-velinov) [reply CONTRIBUTING.md:73]: Works for me

File: CONTRIBUTING.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (theghost5800) [reply CONTRIBUTING.md]: Yes, it is working. Maybe I didn't copy the whole link.

**Review by theghost5800 — APPROVED** [review]

**Pull request merged**